keyboards/converter/modelm_ssk/keymaps/default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2021 tiltowait
```

Comment:
This file should be removed.

Reply:
Resolved in 1f73bf5f97c26508e831d90d8fe610dcfc78e3e4.